### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by lincoln1883**
In this milestone I have done:
- Added linters and config files to the project setup.
- Added HTML structure to replicate the design.
- Added styling to the page structure.
- Added markup to the HTML to refine.
- Add my first build.
- Edit typos and fix linting errors.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+}
+
+h1,
+h2 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 900;
+  margin: 0;
+}
+
+h1 {
```
Comment (thecodechaser) [inline src/style.css]: - [ ] Please make sure to fix the below style issues to match the wireframe design.
  - Please make sure the Leaderboard heading is aligned at the top of the scores section. ( On big screens it's displaying on the left side, It should be visible at the top of the scores section on every screen).

### Your design

image

### Wireframe design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @lincoln1883👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Highly organized code. ✔️
- No linter errors. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (lincoln1883) [reply src/style.css]: I don't mean to kill your joy but your wide-screen monitor is over-exaggerating! however, ill make the necessary changes thank you.

**Review by DuaneDave — APPROVED** [review]
Hi @lincoln1883 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Your design looks good

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**